project_recalculate/tests/test_project_project.py

```diff
@@ -11,14 +10,14 @@ class TestProjectProjectBegin(base.BaseCase):
     project_final_dates_one_task = {
         'date_begin': [
             # name, date_start, date
-            ['pj_0', '2015-08-01', '2015-08-05'],
-            ['pj_1', '2015-08-02', '2015-08-05'],
-            ['pj_2', '2015-08-03', '2015-08-05'],
+            ['pj0', date(2015, 8, 1), date(2015, 8, 5)],
```

Comment:
Why changing key names?

Reply:
because if this module is tested with the project_key module, then the tests do not pass. The project_key module adds the `key` field at the project level and the value of that field must be unique. What happens is that this field is automatically filled depending on the module name and for these module names, the value generated for the `key` field is the same.

Reply:
This must be definitively fixed on `project_key` which is causing a lot of problems with the rest of the modules. Please do another PR disabling the features if on test mode and not testing the module itself (via a context key). Example: https://github.com/OCA/bank-payment/blob/7ce8bbb88bd8c889347a7550aa973124b847af58/account_payment_transfer_reconcile_batch/models/payment_order.py#L28